docs/man/docker-cp.1.md

```diff
@@ -2,69 +2,141 @@
 % Docker Community
 % JUNE 2014
 # NAME
-docker-cp - Copy files or folders from a container's PATH to a HOSTDIR
-or to STDOUT.
+docker-cp - Copy files/folders between containers and the local filesystem.
```

Comment:
`Copy files/folders between a container and the local file system.`  Single container.